src/platforms/java/common/integrations/quartz.mdx

```diff
@@ -63,3 +63,9 @@ trigger.setJobDataAsMap(Collections.singletonMap(SENTRY_SLUG_KEY, "monitor_slug_
 Setting the monitor slug on the `Trigger` will override what is set on the `JobDetail`. This also means you can set up a separate monitor per `Trigger`.
 
 </Note>
+
+<Note>
+
+In case you are using Spring Boot, have defined a `SchedulerFactoryBeanCustomizer` and are setting a global job listener you have to manually add a new `SentryJobListener` instance to the global job listeners as well.
```

Comment:
Not sure if that is better, but I feel like the sentence should be a little easier to digest :)
All good from a technical perspective though.

Setting a global job listener using a `SchedulerFactoryBeanCustomizer` in Spring Boot, will override the listener set by Sentry. For Sentry to work properly, you need to add a new `SentryJobListener` instance to the global job listeners.